CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # App Center SDK for iOS, macOS and tvOS Change Log
 
+## Version 2.5.3 (Under development)
+
+### App Center
+
+* **[Fix]** Optimization of release objects from memory when add an item to a channel.
```

Comment:
channel is internal thing so this message says nothing to customers, please rephrase.